packages/react-devtools-shared/src/__tests__/storeStressTestConcurrent-test.js

```diff
@@ -419,7 +415,7 @@ describe('StoreStressConcurrent', () => {
         ),
       );
       // We snapshot each step once so it doesn't regress.d
-      snapshots.push(print(store));
+      snapshots.push(print(store, false, null, false));
```

Comment:
These are flipping between `null` rects and `[]` rects. Need to double check if intentional and if we can avoid that by rewriting tests.